ktxspec.adoc

```diff
@@ -955,19 +960,22 @@ A _dfDescriptor_ is useful in the following cases:
 The following restrictions must be obeyed when setting the fields of a
 _dfDescriptorBlock_.
 
-* If <<_vkformat,`vkFormat`>> is not `VK_FORMAT_UNDEFINED`, the DFD's
-  `texelBlockDimension*`, `bytesPlane*` and sample information fields
-  must match the format's definition. The `colorModel` must be
-  `KHR_DF_MODEL_RGBSDA`, `KHR_DF_MODEL_YUVSDA` or the matching block
-  compressed color model listed in <<KDF13>> Section 5.6 or its
-  successors, currently `KHR_DF_MODEL_BC1A` to `KHR_DF_MODEL_UASTC`.
+* If <<_vkformat,`vkFormat`>> is not `VK_FORMAT_UNDEFINED`, the
+  DFD's `texelBlockDimension*`, `bytesPlane*` and sample information
+  fields must match the format's definition. The `colorModel` must
+  be `KHR_DF_MODEL_RGBSDA`, `KHR_DF_MODEL_YUVSDA` or the matching
+  block compressed color model listed in <<KDF14>>
+  {url-df-spec}#CompressedFormatModels[Section 5.6] or its successors,
+  currently `KHR_DF_MODEL_BC1A` to `KHR_DF_MODEL_UASTC`.
   `KHR_DF_MODEL_YUVSDA` should be used for all non-prohibited
   `+*_422_*+` formats.
 * If <<_vkformat,`vkFormat`>> is one of the `+*_SRGB{,_*}+` formats,
   `transferFunction` must be `KHR_DF_TRANSFER_SRGB`.
 * If <<_vkformat,`vkFormat`>> is not one of the `+*_SRGB{,_*}+` formats
   and an sRGB variant of that format exists, `transferFunction` should not
   be `KHR_DF_TRANSFER_SRGB`.
+* `transferFunction` must not be `KHR_DF_TRANSFER_HLG_UNNORMALIZED_OETF`.
+  This transfer function is no longer part of the HLG specification.
```

Comment:
There is no technical reason to disallow this enum. If used (for whatever reason) it would mean that slightly different constants were used during the encoding and the scene-referred values were in the [0:12] range.

Tools may warn that it's officially deprecated though.

Reply:
Given that only sRGB and linear were supported in KTX before and that it is not in the current HLG spec, why start allowing it now? It won't break anything. People can use an image processing tool to convert their images when creating KTX files.

Reply:
The technical work is already done.

Reply:
It's not about the KTX-Software repo. _Every_ conformant KTX implementation would have to explicitly reject this single enum value although there is nothing really special about it.

FWIW, the KTX spec allows two (!) legacy PAL functions, which are arguably even less useful than unnormalized HLG.

Reply:
For completeness of the argument:

- `KHR_DF_TRANSFER_NTSC` is legacy and normatively ambiguous (see the complete description in KDFS and the list of references)
- `KHR_DF_TRANSFER_PAL_OETF` is also legacy
- `KHR_DF_TRANSFER_PAL625_EOTF` is legacy, deprecated, and commented as "unrealistic"
- `KHR_DF_TRANSFER_ST240` is legacy and deprecated in favour of BT.709

Yet all these transfer functions are allowed.

Reply:
Maybe we should prohibit those too! :-) Just joking. I am concerned about viewers having to support all these. I realize one more is not going make much difference to the problem. I think it is okay for viewers to output a note that the file uses an unsupported transfer function so colors may be  off.

Reply:
> I think it is okay for viewers to output a note that the file uses an unsupported transfer function so colors may be off.

Even if a viewer diligently implements all conversions, a user would need a display with the widest possible gamut and enough precision to actually see the correct colors. For instance, NTSC colors cannot be shown on a BT.709 display (although this thread is not about primaries, they are typically used together with the corresponding TFs).

I agree that tools (not the spec) should output some warnings/infos about these TFs.

Ironically, viewing `HLG_UNNORMALIZED` content is no different than viewing regular HLG content as the main difference lies in scene-referred values that are not intended for display anyway.

Reply:
I removed the prohibition in https://github.com/KhronosGroup/KTX-Specification/pull/224/commits/c50ec229be1fa463691cb3271822d7a994f54b98. Are you ready to re-approve? I want to merge this and request approval from the WG to publish.